docs/setup.md

```diff
@@ -35,10 +35,12 @@ See the example snippet below:
 ```kotlin
 import com.example.something // Imports at the top, as usual.
 
+pluginManagement {} // Optional
+
 buildscript {
    // We will setup refreshVersions here, see below.
 }
-pluginManagement {} // Optional
+
 plugins {} // Optional
```

Comment:
Are you sure this is the correct order? Why not put this change in another PR? It's not really related to buildSrcLibs.

Reply:
Yes I'm sure, here is what happens if I use the documented order:

```
$ ./gradlew help
> /Users/jmfayard/IdeaProjects/refreshVersions/sample-kotlin/settings.gradle.kts:3:1: 
Unexpected `buildscript` block found. `buildscript` can not appear before `pluginManagement`.
```